apps/web/components/Charts/ChartsCard/ChartsCard.tsx

```diff
@@ -97,23 +78,24 @@ export const ChartsCard: React.FC<ChartsCardsProps> = ({ data, blue }) => {
               <Text color="dark400">{graphDescription}</Text>
             )}
           </Box>
+
+          {subPage && (
+            <Box padding={[2, 2, 4]}>
+              <ExportCSVButton data={graph.data} title={graph.title} />
+            </Box>
+          )}
         </Box>
-        {blue && (
-          <Box padding={[2, 2, 4]}>
-            <ExportCSVButton data={graph.data} />
-          </Box>
-        )}
       </Box>
-
       <Box
         display="flex"
         justifyContent="center"
         alignItems="center"
-        style={{ width: '100%', height: '100%' }}
+        style={{
+          width: graph.type === 'Pie' || !shouldStack ? '100%' : '889px',
+          height: '100%',
+        }}
       >
-        <Box justifyContent="center" style={{ width: '80%', height: '408px' }}>
-          {children}
-        </Box>
+        <Box className={cn(styles.graphParent)}>{children}</Box>
```

Comment:
only use classnames library when working with multiple classNames
```suggestion
        <Box className={styles.graphParent}>{children}</Box>
```

Reply:
this applies to all instances